ProcessMaker/spark
# Session refresh modal

closes #1524 

How to test:
change session.php 'lifetime' => 120, to 'lifetime' => 1
refresh page and wait one minute for the modal.
Clicking continue session will restart the session.

This PR also refreshes the session in both the modeler and the screen builder when changes are made.
To test this go to either page with your lifetime still set to 1 minute. Make changes and you will see that the session modal does not come up.

The session refresh modal happens across all tabs open. 
![Screen Shot 2019-04-11 at 10 33 58 AM](https://user-images.githubusercontent.com/29641725/55978721-56301000-5c45-11e9-8455-4e909e1a30c9.png)
